ansible_rulebook/rule_set_runner.py

```diff
@@ -430,10 +431,18 @@ async def _call_action(
                     var_root = action_args.pop("var_root")
                     _update_variables(variables_copy, var_root)
 
-                action_args = {
-                    k: substitute_variables(v, variables_copy)
-                    for k, v in action_args.items()
-                }
+                if action == "debug":
```

Comment:
@zkayyali812 Should this include print_event too.

Reply:
I left it out and handle print_event directly here - https://github.com/zkayyali812/ansible-rulebook/pull/3/files#diff-678eff0fb8b65ef65626feb4806dc3a6674306e2cd14ab0e09ebc094cb7d6992R48

When tested with `print_event` here, there is no effect, unless it is specified in the action directly

Reply:
After further discussion, we decided to not filter on print_event